app/relay.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "src": "./",
+  "language": "typescript",
+  "artifactDirectory": "./__generated__",
+  "customScalars": {
+    "Datetime": "string",
+    "JSON": "any",
+    "BigFloat": "string",
+    "Upload": "any"
+  },
+  "exclude": [
+    "**/.next/**/*",
+    "**/node_modules/**/*",
+    "**/__generated__/**/*",
+    "**/server/**/*"
+  ],
+  "noFutureProofEnums": true,
+  "schema": "./schema/schema.graphql"
```

Comment:
the new relay compiler doesn't like it when the schema is in one of the excluded folders